src/cnxml/deserialize.js

```diff
@@ -137,6 +138,7 @@ const MARK_TAGS = {
     sub: 'subscript',
     sup: 'superscript',
     link: xref,
+    code: 'code',
 }
```

Comment:
Properties in this object should be sorted alphabetically (yep I forgot about it when adding `link`).